frontend/src/app/components/employee/modals/change-passw-modal/change-passw-modal.component.ts

```diff
@@ -7,18 +7,21 @@ import { ApiCommunicationService } from 'src/app/services/api-communication.serv
 
 @Component({
   selector: 'app-change-passw',
-  templateUrl: './change-passw.component.html',
-  styleUrls: ['./change-passw.component.css']
+  templateUrl: './change-passw-modal.component.html',
+  styleUrls: ['./change-passw-modal.component.css']
 })
-export class ChangePasswComponent implements OnInit {
+export class ChangePasswModalComponent implements OnInit {
   
-  public firstPass: any;
-  public secondPass: any;
+  public firstPassword: any = "firstPassword";
```

Comment:
lehetne ez string? és miért inicializálod miután deklaráltad
```suggestion
  public firstPassword: string;
```

Reply:
a modallal próbálgattam, elfelejtettem kivenni